app/helpers/reactjs_helper.rb

```diff
@@ -0,0 +1,8 @@
+module ReactjsHelper
+
+  def mount_react_component(name, selector, data)
```

Comment:
Why not push the placeholder into the helper as well, and create a random ID based on the component name to ensure its unique instead of passing a selector? somthing like:
```ruby
  def mount_react_component(name, data)
    id = "#{name}-#{rand(100000)}"
    content_tag(:div, :id => id) +
    javascript_tag defer: 'defer' do
      "$(reactMounter.mount('#{name}', '##{id}', #{data}));".html_safe
    end
  end
```